config/gregtech/worldgen/vein/overworld/sulfidic_silver_vein.json

```diff
@@ -10,8 +10,8 @@
   "generator": {
```

Comment:
we could also change galena to proustite since galena has its own vein

Reply:
Doesn't proustite have it's own vein too?